cmd/renterd/config.go

```diff
@@ -97,9 +97,10 @@ func defaultConfig() config.Config {
 		Worker: config.Worker{
 			Enabled: true,
 
-			ID:                  "worker",
-			ContractLockTimeout: 30 * time.Second,
-			BusFlushInterval:    5 * time.Second,
+			ID:                     "worker",
+			AccountsRefillInterval: defaultAccountRefillInterval,
```

Comment:
Do we need to reconsider the `defaultAccountRefillInterval`? I always found it to be quite on the low side.

Reply:
I think it's probably fine. We don't really do anything unless we reach a certain min balance or force a sync.